src/streamvbyte.c

```diff
@@ -155,11 +220,22 @@ size_t streamvbyte_encode(uint32_t *in, uint32_t count, uint8_t *out) {
   uint8_t *keyPtr = out;
   uint32_t keyLen = (count + 3) / 4;  // 2-bits rounded to full byte
   uint8_t *dataPtr = keyPtr + keyLen; // variable byte data after all keys
-#ifdef __AVX__
-  return svb_encode_vector(in, keyPtr, dataPtr, count) - out;
-#else
-  return svb_encode_scalar(in, keyPtr, dataPtr, count) - out;
+
+#if defined(__AVX__) || defined(__ARM_NEON__)
```

Comment:
This is an attempt to unify the interface for vector-based encoding in 4-word quads.  Another issue/branch should address breaking these out in an inline/static-friendly way.